keyboards/gmmk/numpad/numpad.c

```diff
@@ -107,7 +107,7 @@ led_config_t g_led_config = {{
 	2, 2, 2, 2, 2, 2, 2
 } };
 
-#    ifdef AW20216S_PW_EN_PIN_1
+#    ifdef AW20216S_PW_EN_PIN
 
 void keyboard_pre_init_user(void) {
```

Comment:
While we are changing this file, please can you update the lines to the following:

```c
void keyboard_pre_init_kb(void) {
    wait_ms(2000);
    gpio_set_pin_output(AW20216S_PW_EN_PIN);
    gpio_write_pin_high(AW20216S_PW_EN_PIN);

    keyboard_pre_init_user();
}
```

Reply:
Sure.

I do not know the code base very well, but shouldn't the user facing initialization `keyboard_pre_init_user()` be called regardless of the definition of `AW20216S_PW_EN_PIN`?

Reply:
There is a default implementation, <https://github.com/qmk/qmk_firmware/blob/master/quantum/keyboard.c#L279-L287>.

This suggested change maintains the expected semantics.

Reply:
Makes sense. Thanks for the hint.